src/edu/century/pa3/TruckOfProduce.java

```diff
@@ -50,9 +45,10 @@ public double computeTotal(TruckOfProduce t) {
 	
 	@Override
 	public String toString() {
-		String str = null;
-		for (int i = 0; i < truck.length; i++)
-			str = str + truck[i].getName();
+		String str = "";
```

Comment:
This is where the null at the beginning of the kiwis was coming from, this is the fix.